actors/power/src/state.rs

```diff
@@ -68,6 +70,8 @@ pub struct State {
     /// Number of miners having proven the minimum consensus power.
     pub miner_above_min_power_count: i64,
 
+    pub epochs_since_ramp_start: u64,
```

Comment:
If I recall our conversation correctly @lemmih - we said that this would be hardcoded to a particular value for each network?

Reply:
please ignore this b/c it is incorrect, at least in the way I was thinking about it, epochs_since_ramp_start needs to be able to be positive or negative, so I've updated this to be i64